docs/conf.py

```diff
@@ -26,7 +26,8 @@
 # NOTE: No obvious way to automatically anticipate all the sub modules without
 # installing the package, which is what we are trying to avoid.
 MOCK_MODULES = [
-    'bokeh', 'bokeh.embed', 'bokeh.resources',
```

Comment:
Why did you remove these?

Reply:
I didn't. I inserted a bunch of bokeh functions in there.